docs/source/audio_dataset_script.mdx

```diff
@@ -0,0 +1,365 @@
+        MultilingualLibrispeechConfig(name="german", description="German LibriSpeech dataset"),
+        MultilingualLibrispeechConfig(name="dutch", description="Dutch LibriSpeech dataset"),
+        MultilingualLibrispeechConfig(name="french", description="French LibriSpeech dataset"),
+        MultilingualLibrispeechConfig(name="spanish", description="Spanish LibriSpeech dataset"),
+        MultilingualLibrispeechConfig(name="italian", description="Italian LibriSpeech dataset"),
+        MultilingualLibrispeechConfig(name="portuguese", description="Portuguese LibriSpeech dataset"),
+        MultilingualLibrispeechConfig(name="polish", description="Polish LibriSpeech dataset"),
+    ]
+```
+
+Now if users want to load the `polish` configuration, they can use the configuration name:
+
+```py
+>>> from datasets import load_dataset
+>>> dataset = load_dataset("facebook/multilingual_librispeech", "polish", split="train")
+```
+
+### Add dataset metadata
+
+Adding information about your dataset helps users to learn more about it. This information is stored in the [`DatasetInfo`] class which is returned by the `info` method. Users can access this information by:
+
+```py
+>>> from datasets import load_dataset_builder
+>>> ds_builder = load_dataset_builder("vivos")
+>>> ds_builder.info
+```
+
+There is a lot of information you can specify about your dataset, but some important ones to include are:
+
+1. `description` provides a concise description of the dataset.
+2. `features` specify the dataset column types. Since you're creating an audio loading script, you'll need to include the [`Audio`] feature and the `sampling_rate` of the dataset.
+3. `homepage` provides a link to the dataset homepage.
+4. `license` specify the permissions for using a dataset as defined by the license type.
+5. `citation` is a BibTeX citation of the dataset.
+
+<Tip>
+
+You'll notice a lot of the dataset information is defined earlier in the loading script which makes it easier to read. There are also other [`~Datasets.Features`] you can input, so be sure to check out the full list for more details.
+
+</Tip>
+
+```py
+def _info(self):
+    return datasets.DatasetInfo(
+        description=_DESCRIPTION,
+        features=datasets.Features(
+            {
+                "speaker_id": datasets.Value("string"),
+                "path": datasets.Value("string"),
+                "audio": datasets.Audio(sampling_rate=16_000),
+                "sentence": datasets.Value("string"),
+            }
+        ),
+        supervised_keys=None,
+        homepage=_HOMEPAGE,
+        license=_LICENSE,
+        citation=_CITATION,
+    )
+```
+
+### Download and define the dataset splits
+
+Now that you've added some information about your dataset, the next step is to download the dataset and define the splits.
+
+1. Use the [`~DownloadManager.download`] method to download and extract the given URLs. The URLs are replaced with a path to the local files. This method accepts:
```

Comment:
first of all, vivos is not available for now :( 

secondly, I've been thinking on how to explain things in an easy but still correct way and ended up realizing that it's actually not possible! :D so I have the following suggestions:
1. use https://huggingface.co/datasets/indonesian-nlp/librivox-indonesia/blob/main/librivox-indonesia.py as an example for now. it's clean and small in size and has both streaming support and local file paths. what do you think? cc @lhoestq 
2. explain in detail, like, really in detail, what's going on here in `_split_generators` and `_generate_examples`. I've written a dirty draft on how it can be done [here](https://gist.github.com/polinaeterna/fe0f95d22b90813fe7e880a405cc0013).
audio datasets authors who took the risk to write their own dataset script should be advanced users so I believe it's worth explaining everything just as is straightaway!  

let me know what you think! and if you agree on that, @stevhliu feel free to integrate my notes linked in this comment, rewording them as you feel would be better, and also don't hesitate to ping me on slack if you need a quick check or smth, I'll be happy to help! ❤️

Reply:
another suggestion from @lhoestq !
we first describe `vivos`-like approach (without extraction), and then add a section which explain how to also make your dataset extracted locally and preserve full paths to local files (with the example I suggested). but that way we need to fix `vivos` first (we are waiting for their response)

Reply:
I went with suggestion #1 to replace `vivos` with `librivox/indonesia` so we can have some nice examples right away without waiting for `vivos` to be fixed. 

Thanks again for your super detailed notes! I think I've integrated all of them, but please let me know what you think and what can be improved for even more clarity 🥰

Reply:
Hi @stevhliu, the documentation looks very good, and thanks for using my dataset as an example, although I just created it a few days ago :-) And since it is new, I might still need to update a few small things, such as the STATS in release_stats.py. I think this will not affect the documentation you are writing. However, I also updated how the 
compressed tar file is stored. I split the tar file into audio_train.tgz and audio_test.tgz to get the test data directly if I load the test split in streaming mode. Otherwise, the loading process will first scan all the train files before it arrives to test files which could take some time if the dataset is huge.